modernomad/core/tests/test_emails.py

```diff
@@ -44,7 +44,7 @@ class MockResponse():
         self.admin = self.create_user('admin1', admin=True, email='admin1@bob.com')
         self.booking = self.create_booking(user=self.guest1)
 
-        today = datetime.now().date()
+        today = timezone.localtime(timezone.now())
```

Comment:
If the test depends on time it would be better to mock the time, that way the time is always the same when the test runs. In general all across the application it should be using `timezone.now()` rather than datetime. 

What is it that `timezone.localtime` does in this case?

Reply:
Sorry yeah I should have been clearer -- this is an attempt at a quick fix to make the tests work. No idea what it is I just copied it from the implementaiton.

Correct solution is https://github.com/spulec/freezegun

Feel free to spend more time on this if you want -- I haven't got the cycles. :)